rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -687,8 +777,8 @@ int wait4child(pal_exec_args *attr)
 }
 /* *INDENT-ON* */
 
-int __pal_get_local_report(void *targetinfo, int targetinfo_len, void *report,
-			   int *report_len)
+int __pal_get_local_report(uint64_t base, void *targetinfo,
```

Comment:
Please sync parameters with api_v3_linux.go as below: other question, How and from where to pass the parameter "base"? 
static int palGetLocalReport(void *sym, void *target_info, int target_info_len,
 22                                                         void *report, int* report_len)
 23 {
 24         return ((int (*)(void *, int, void*, int*))sym)(target_info, target_info_len,
 25                                                                 report, report_len);
 26 }

Reply:
I screwed up it. Rework.